elements/tcpudp/settcpchecksumip6.cc

```diff
@@ -0,0 +1,71 @@
+/*
+ * settcpchecksumip6.{cc,hh} -- sets the TCP header checksum for IPv6 packets
+ * Robert Morris
```

Comment:
Can you add your name and copyright institution? Maybe add "based on settcpchecksum.{cc,hh}"

Reply:
Yes, of course. I didn't want to claim too much credit, but the way you suggest it is a good way, referencing the others. Pushed!